thesis-de/content-de.tex

```diff
@@ -2807,6 +2809,104 @@ \section{Realisierung Grafana}
 erstellt~\cite{grafana-issue}.
 \mr%
 
+\subsection{Graphen definieren}
+\label{subsec:graphen-definieren}
+Soabld die Grundkonfiguration von Grafana wie in~\ref{sec:realisierung_grafana}
+beschrieben durchgeführt worden ist, können als nächstes Graphen visualisiert
+werden. Zu Beginn wird hier unter dem Punkt \texttt{General} ein
+aussagekräftiger Titel, sowie Beschreibung des Graphen hinterlegt. In dem
+Beispiel, welches im Anhang~\ref{figure:grafana_general} enthalten wird, wird
+der Graph für die Prozessor Auslastung definiert. Unter dem Punkt
+\texttt{General} können alle Einstellungen für das Aussehen des Graphen
+hinterlegt und angepasst werden. Ebenfalls können Verlinkungen zu anderen
+Dashboards oder direkten URL Adressen hinterlegt werden. Diese werden
+anschließend, wie im Anhang~\ref{figure:grafana_description}  dargestellt. Dies
+dient zur besseren Übersichtlichkeit, wenn ein Administrator oder die
+Führungsebene diese Graphen analysieren möchten. Im zweiten Schritt werden nun
+die einzelnen Graphen definiert.
+
+Hierzu muss unter \texttt{Metrics} zunächst ausgewählt werden, von welcher
+Datenquelle die erhaltenen Daten stammen und anschließend über \texttt{Add
+Query} hinzugefügt werden. Hier wird zunächst eine Art \texttt{SELECT}
+\gls{SQL}-Abfrage hinterlegt, welche für die entsprechende Anforderung
+angepasst werden kann. Innerhalb der SQL-Abfrage wird definiert, von wo die
+erhaltenen Daten stammen, sowie welche Werte abgerufen werden sollen. Ebenfalls
+kann diese SQL Abfrage mit anderen SQL Abfragen innerhalb einer Gruppe zusammen
+definiert werden. In dem Projekt wird für die Daten Erhaltung eine
+API~\gls{API} Schnittstelle verwendet, welche im Laufe des Projektes entwickelt
+und entworfen worden ist. Die API liefert hier einen \gls{JSON}-Wert zurück.
+Damit Grafana diesen als Datenquelle verwenden kann, muss hier zuvor noch ein
+Plugin installiert werden. Es stehen viele verschiedene Plugins auf der Grafana
+Webseite zur Verfügung. Zu jedem Plugin gibt es hier eine kurze Erläuterung,
+sowie eine step-by-step Anleitung zur installation dieses Plugins. Die Plugins
+werden hier über einen \texttt{bash} Befehl installiert. Anschließend kann
```

Comment:
genau genommen wird das kommandozeilenprogramm in der bash ausgeführt, aber welche shell das ist interessiert bestimmt keinen.